src/Controller/ModulesController.php

```diff
@@ -158,6 +159,7 @@ public function view($id) : ?Response
 
             return $this->redirect(['_name' => 'modules:list', 'object_type' => $this->objectType]);
         }
+        $this->ProjectConfiguration->read();
```

Comment:
... something like `@property ProjectConfigurationComponent ProjectConfiguration The project configuration component`?

Reply:
Right, I'll add it